tests/models/test_candidates.py

```diff
@@ -283,6 +283,7 @@ def test_ignore_invalid_py_version(project):
     _ = next(iter(repo.find_candidates(req)))
 
 
+@pytest.mark.network
 def test_find_candidates_from_find_links(project):
     project.pyproject.settings["source"] = [
         {"name": "test", "url": "http://fixtures.test/index/demo.html", "verify_ssl": False, "type": "find_links"}
```

Comment:
```suggestion
        {"name": "pypi", "url": "http://fixtures.test/index/demo.html", "verify_ssl": False, "type": "find_links"}
```

Can you verify this change fixes the network problem?

Reply:
Thanks, confirmed it does and updated the pull request.